src/send/EnterAmount.tsx

```diff
@@ -477,8 +477,7 @@ const styles = StyleSheet.create({
     ...typeScale.labelSemiBoldMedium,
   },
   warning: {
-    marginTop: Spacing.Regular16,
-    marginBottom: Spacing.Smallest8,
+    marginBottom: Spacing.Regular16,
```

Comment:
also no visual changes here, this ensures that every element is applying their own bottom spaces, so that the jumpstart warning does not need to apply top spacing